src/Execution/ModelNotFoundErrorHandler.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Nuwave\Lighthouse\Execution;
+
+use Closure;
+use GraphQL\Error\Error;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+
+/**
+ * Wrap model not found exceptions.
+ */
+class ModelNotFoundErrorHandler implements ErrorHandler
+{
+    public function __invoke(?Error $error, Closure $next): ?array
+    {
+        if (null === $error) {
+            return $next(null);
+        }
+
+        $previous = $error->getPrevious();
+
+        if ($previous instanceof ModelNotFoundException) {
+            return $next(new Error($previous->getMessage(), null, null, [], null, null, ['category' => 'not_found']));
```

Comment:
```suggestion
            return $next(new Error($previous->getMessage(), null, null, [], null, $previous));
```

1. Include `$previous`
2. `category` will be removed in the next major version anyways

Reply:
One minor question about removing `category` in next version, will there be a replacement to check for different types of errors?

I use for example the `category` **validation** to detect validation errors and show them differently than other errors.

Reply:
https://github.com/webonyx/graphql-php/blob/master/UPGRADE.md#breaking-removed-error-extension-field-category

I am not going to re-add it in Lighthouse.

Reply:
No problem I understand, I'll remove it from my implementation :)